src/qt/locale/bitcoin_nl.ts

```diff
@@ -4495,11 +4495,11 @@ Use this functionality with extreme caution.</source>
     </message>
     <message>
         <source>Execute command when a wallet transaction changes (%s in cmd is replaced by TxID, %i with block height, with a value of 0 if tx is no longer in chaintip)</source>
-        <translation type="unfinished"></translation>
+        <translation>Voer een commando uit wanneer een wallet transactie veranderd (%s in cmd is replaced by TxID, %i with block height, with a value of 0 if tx is no longer in chaintip)</translation>
```

Comment:
Perhaps: 

(%s in commando wordt vervangen door TxID, %i met de block hoogte, met een waarde van 0 wanneer de transactie niet in de blockchain voorkomt)

Reply:
Ah right, good proposal! Small edit

(%s het commando wordt vervangen door TxID, %i met een block hoogte, met een waarde van 0 als de transactie niet in de blockchain voorkomt)

Do i need to make a new merge request?

Reply:
> Do i need to make a new merge request?

No, just edit and save it in the same branch on your repo and it will show up here.

Reply:
Okay done :)